swap_meet/vendor.py

```diff
@@ -1,2 +1,80 @@
+from curses import wrapper
+from re import T
+from signal import set_wakeup_fd
+from pytest import Item
+from swap_meet.item import Item 
```

Comment:
I would check to make sure all of these imports are being used, if not delete them.